apparmor.d/groups/kde/kwin_wayland

```diff
@@ -78,10 +95,20 @@ profile kwin_wayland @{exec_path} flags=(attach_disconnected mediate_deleted) {
   @{run}/udev/data/c13:@{int}  r,         # for /dev/input/*
   @{run}/udev/data/c226:@{int} r,         # for /dev/dri/card*
 
+  @{run}/udev/data/+usb:* r,
```

Comment:
Can you sort all `udev/data` as it is usually sorted:
1. `udev/data/+`
2. `udev/data/c`

Also try to put the same comment you can find elsewhere. Eg:
```
  @{run}/udev/data/c189:@{int} r,         # for /dev/bus/usb/**
```